docs/other/model-in-memory.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: model_in_memory
+title: Creating AMF model in memory
+---
+:::info
+The functionality expressed in this documentation requires familiarity with the
+[amf model](https://github.com/aml-org/amf/blob/develop/documentation/model.md).
+:::
+
+AMF is most commonly used to parse an already existing API,
+but it can also be used to create an API in a fluent and descriptive way.
```

Comment:
AMF allows you to parse existing APIs, but you can also use it to model fluent, descriptive APIs.

OR

AMF allows you to parse existing APIs, but you can also use it to model fluent, descriptive APIs in memory.

Is the "in memory" part important here or in the following sentence?